stdimage/management/commands/rendervariations.py

```diff
@@ -13,14 +12,22 @@
 
 from stdimage.utils import render_variations
 
+try:
+    import resource
+except ImportError:
+    resource = None
+
+
 BAR = None
 
 
 class MemoryUsageWidget(progressbar.widgets.WidgetBase):
     def __call__(self, progress, data):
-        return 'RAM: {0:10.1f} MB'.format(
-            resource.getrusage(resource.RUSAGE_SELF).ru_maxrss / 1024
-        )
+        if resource is not None:
```

Comment:
Oh alright. It'll be some hours before I'll be able to switch those around (day job). Though, the your first suggestion is not necessarily incorrect since it's a more common branch, likely to be hit more often in real life. Let me know if you still want them flipped.